### PR TITLE
Fix to website

### PR DESCRIPTION
**Pull request by jaeilepp**
Fixed the links for API.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
You shouldn't need to do this to build the docs. I don't need to. It's also a bit misleading, because a lot of these things exist in the root module `mne.` but they're here referenced from e.g. `mne.io.fiff.` or so.

File: doc/python_reference.rst [hunk 1]
```diff
@@ -565,28 +574,14 @@ Functions:
...
-   read_trans
-   read_source_spaces
-   read_surface
-   sensitivity_map
-   setup_source_space
-   setup_volume_source_space
-   write_bem_surface
-   write_trans
+   _make_forward.make_forward_solution
+   _field_interpolation.make_field_map
```
Comment (agramfort) [inline doc/python_reference.rst]: don't document private code

**Comment by agramfort** [comment]
indeed it should not be necessary. I don't need this either.

**Comment by jaeilepp** [comment]
Well the links from ``__init__`` just don't seem to work for me no matter what I do.

**Comment by Eric89GXL** [comment]
Try upgrading Sphinx maybe? @agramfort are there other packages that might need to be upgraded?

You running Anaconda, or something else?

**Comment by agramfort** [comment]
what sphinx version are you using?

**Comment by jaeilepp** [comment]
Version 1.3.1

**Comment by jaeilepp** [comment]
And I'm using anaconda.

**Comment by agramfort** [comment]
I do have the same problem with anaconda + sphinx 1.3.1

not sure what's wrong but it used to work fine without exposing the internals of full imports.

**Comment by jaeilepp** [comment]
Yeah, it seems to generate the rst files with full path and then tries to link them with the condensed path and fails.

**Comment by jaeilepp** [comment]
After hours of debugging I found that it's Sphinx bug related to https://github.com/sphinx-doc/sphinx/pull/1892.

Building the html now...

**Pull request closed without merging**